app/controllers/tasks.js

```diff
@@ -33,11 +34,44 @@ export default class TasksController extends Controller {
   @tracked message = ''; // this is required in the modal
   @tracked buttonRequired = false; // this is required in the modal
```

Comment:
Are these comments required? If yes, can we please make them more short and crisp
For e.g. : // Required for modal 
or 
// Required for task holder

Reply:
I don't think they are really needed